patternlab/styleguide/source/_patterns/01-atoms/09-media/figure.twig

```diff
@@ -1,16 +1,14 @@
-{% set figureClass = figure.align ? "ma__figure--" ~ figure.align : "" %}
-
-<figure class="ma__figure {{ figureClass }}">
-  {% set image = figure.image %}
+{% set alignmentClass = figure.align ? " ma__figure--" ~ figure.align : "" %}

```

Comment:
Should we also add a `sizeClass` calculation here?  My understanding is that we want to be able to apply consistent size classes to tableau embeds to start with, but that images and iframes will be coming soon. If that's right, the `figure` template could accept a `size` argument that allows you to set the size:
```twig
{% set sizeClass = figure.size ? " ma__figure--" ~ figure.size : "" %}
```
Then, below, we'd print it:
```twig
<figure class="ma__figure{{ alignmentClass }}{{ sizeClass }}{{ figure.class ? " "~ figure.class : "" }}">
```

Reply:
Implemented in #971.